pkg/resourcetracker/app.go

```diff
@@ -100,17 +106,42 @@ func CreateComponentRevisionResourceTracker(ctx context.Context, cli client.Clie
 	return createResourceTracker(ctx, cli, app, getComponentRevisionResourceTrackerName(app), v1beta1.ResourceTrackerTypeComponentRevision)
 }
 
-// ListApplicationResourceTrackers list resource trackers for application with all historyRTs sorted by version number
-func ListApplicationResourceTrackers(ctx context.Context, cli client.Client, app *v1beta1.Application) (rootRT *v1beta1.ResourceTracker, currentRT *v1beta1.ResourceTracker, historyRTs []*v1beta1.ResourceTracker, crRT *v1beta1.ResourceTracker, err error) {
-	metrics.ListResourceTrackerCounter.WithLabelValues("application").Inc()
+func listApplicationResourceTrackers(ctx context.Context, cli client.Client, app *v1beta1.Application) ([]v1beta1.ResourceTracker, error) {
 	rts := v1beta1.ResourceTrackerList{}
-	if err = cli.List(ctx, &rts, client.MatchingLabels{
+	err := cli.List(ctx, &rts, client.MatchingLabels{
 		oam.LabelAppName:      app.Name,
 		oam.LabelAppNamespace: app.Namespace,
-	}); err != nil {
+	})
+	if err == nil {
+		return rts.Items, nil
+	}
+	if !kerrors.IsForbidden(err) && !kerrors.IsUnauthorized(err) {
+		return nil, err
+	}
```

Comment:
```suggestion
	}
	
	// we will check the apiextension server for unauthorized apps
```

Let's add some explanation here for more clear